src/intents.cs

```diff
@@ -2294,6 +2806,13 @@ interface INInteraction : NSSecureCoding, NSCopying {
 
 		[NullAllowed, Export ("groupIdentifier")]
 		string GroupIdentifier { get; set; }
+
+		// From INParameter.h INInteraction ()
+
+		[iOS (11,0), Watch (4,0), NoMac]
+		[Export ("parameterValueForParameter:")]
+		[return: NullAllowed]
+		NSObject GetParameterValue (INParameter parameter);
```

Comment:
We can constrain it to `NSObject` for now and if needed expand it to `INativeObject` in the future